README.md

```diff
@@ -168,6 +168,60 @@ Por convenção utilizados o seguintes campos dentro do payload de erro:
 - **reason**: mais detalhes do erro
 - **responsecode**: código do erro
 
+## Detalhamento dos atributos
+
+### Payment Request (Pagamento)
+
+`PaymentRequest.java`
+| Atributo                 | Tipo            |  Exemplo                         | Descrição                     | 
+| ------------------------ |:--------------: | :------------------------------: | ----------------------------- |
+| installmentsInterestRate | String          | 0.05                             | Taxa de juros de parcelamento |
```

Comment:
In the past, it was like `5%` instead of `0.05`. Did we improve it?

Reply:
There's no integration using this, we can rewrite to use a better form.